cmd/virt-handler/virt-handler.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+# Configure libvirt username and password
+sed -i s/^authname=/authname=$(cat /etc/sasl-secret/username)/g /etc/libvirt/auth.conf
+sed -i s/^password=/password=$(cat /etc/sasl-secret/password)/g /etc/libvirt/auth.conf
```

Comment:
would be easier to just generate that file.
Or virt-handler can use it natively if you keep(and enhance) the "--libvirt-uri" parameter, then no need for these files....